webapp/src/Controller/API/JudgehostController.php

```diff
@@ -812,37 +810,47 @@ public function internalErrorAction(Request $request)
 
         if (in_array($disabled['kind'], ['problem', 'language', 'judgehost']) && $judgingId) {
             // Give back judging if we have to.
-            $this->giveBackJudging((int)$judgingId);
+            $hostname = $request->request->get('hostname');
+            /** @var Judgehost $judgehost */
+            $judgehost = $this->em->createQueryBuilder()
```

Comment:
why not use `getRepository->find` here?

Reply:
Copy paste from somewhere else, that sounds better